modules/server/src/main/scala/higherkindness/compendium/storage/FileStorage.scala

```diff
@@ -41,10 +41,10 @@ object FileStorage {
           }
         } yield ()
 
-      override def recover(id: ProtocolId): F[Option[Protocol]] =
+      override def recover(metaProtocolDB: MetaProtocolDB): F[Option[MetaProtocol]] =
```

Comment:
We are lifting the refined restriction here for protocol identifiers, is there any way to get it back or it isn't worth the effort?

Reply:
What do you mean? I think I don't see your point

Reply:
Previously we required the protocol identifier to be refined (by `ProtocolId`). Using `MetaProtocolDB` removes this restriction, as it is a `String` in `final case class MetaProtocolDB(idlName: IdlNames, id: String)`.

Reply:
Oh yeah, I see your point now 😂 

Yes, we can (should) use the refined type indeed. I set it as String because when I started to work in this branch, the type `ProtocolId` didn't exist. So yeah, no problem to move from `String` to `ProtocolId`

Reply:
I agree, we should move `MetaProtocolDB` to `Server`

Reply:
It is on the server package already